functionary/ui/templates/account/signup_closed.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Functionary</title>
+        <meta http-equiv="refresh" content="0;URL={% url 'account_login' %}" />
+    </head>
+    <body>
+        <h1>Account signup is disabled.</h1>
```

Comment:
Similar question asked elsewhere:  How does one get to this page? Should it just go away?